docs/DeveloperGuide.md

```diff
@@ -514,18 +523,20 @@ This is a quick overview of all the commands and their functionalities.
 
 **Target user profile**:
 * hotel manager of a small-sized hotel
-* hotel manager who want to keep track of guests
+* hotel manager who want to keep track of guests' details
 * has a need to manage a significant number of guests
 * prefer desktop apps over other types
 * can type fast
 * prefers typing to mouse interactions
 * is reasonably comfortable using CLI apps
 
 **Value proposition**:
-* To know when the guests check in and check out
-* To keep track of the guests' bills
-* To note down all the requirements/requests that the guests asked for
-* To note down the status of the room, if the room is cleaned or not cleaned
+* To keep track of the essential information of a guest like name, phone number, email address.
```

Comment:
Should we use "such as the" instead of "like" to sound more formal? Also, should there be an "and" rather than a comma between phone number and email address?

Reply:
Agreed!